src/utils/index.js

```diff
@@ -2,8 +2,18 @@
 import axios from 'axios';
 import jwtDecode from 'jwt-decode';
 
+export const getUrl = (hostName) => {
+  let url;
+  if (hostName.includes('master')) {
+    url = 'https://indiana-ah-master.herokuapp.com/api/v1/';
+    return url;
+  }
+  url = 'https://indiana-ah-staging.herokuapp.com/api/v1/';
+  return url;
```

Comment:
```suggestion
if (hostName.includes('master')) {
    return 'https://indiana-ah-master.herokuapp.com/api/v1/';
  }
  return 'https://indiana-ah-staging.herokuapp.com/api/v1/';
```

Reply:
Why not keep it short as in suggestion above?

Reply:
Noted, I'll do that now